### PR TITLE
Rework header, footer & home page

### PR DESCRIPTION
**Pull request by sandrica89**
* Use header & footer react-dsfr
* New UX for header, footer & home page

### CODE REVIEW

File: frontend/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -1,98 +1,71 @@
...
-            to="/stats"
-            label="Nos statistiques"
-          />
-        </ul>
-      </div>
-    </div>
-  </footer>
+  <DSFooter>
+    <FooterBody description={<FooterDescription />}>
+      <Logo href="https://www.enseignementsup-recherche.gouv.fr/">
```
Comment (sandrica89) [inline frontend/src/components/Footer/Footer.jsx:46]: Le lien ne s'ouvre pas dans une autre fenêtre, non prévu par `<Logo />`

File: frontend/src/components/Header/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { HeaderNav, NavItem } from '@dataesr/react-dsfr';
+
+const defaultItems = [
+  { title: 'Comment ça se passe ?', link: '/' },
+  { title: 'Nos partenaires', link: '/trouver-un-psychologue' },
+  { title: 'Foire aux questions', link: '/faq' },
+  // { title: 'Nous contacter', link: '/mentions-legales' },
```
Comment (sandrica89) [inline frontend/src/components/Header/Nav.jsx:9]: À décommenter quand on aura une vrai page de contact

File: frontend/src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -1,98 +1,71 @@
...
+  FooterBottom,
+  FooterLink,
+  Logo,
+  Link,
+} from '@dataesr/react-dsfr';
 
-import Logo from 'components/Logo/LogoFooter';
-import FooterBottomItem from './FooterBottomItem';
+const footerBodyLinks = [
+  { title: 'Contactez-nous', link: '/mentions-legales' },
```
Comment (desoindx) [inline frontend/src/components/Footer/Footer.jsx:13]: shouldn't we have something to manage the target ? in a new tab or in the same ?

File: frontend/src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -1,98 +1,71 @@
...
-      </div>
-    </div>
-  </footer>
+  <DSFooter>
+    <FooterBody description={<FooterDescription />}>
+      <Logo href="https://www.enseignementsup-recherche.gouv.fr/">
+        Ministère de l&lsquo;Enseignement Supérieur, de la Recherche et de l&lsquo;Innovation
+      </Logo>
+      { footerBodyLinks.map(item => (
+        <FooterBodyItem key={footerBodyLinks.indexOf(item)}>
```
Comment (desoindx) [inline frontend/src/components/Footer/Footer.jsx:50]: don't use index as keys 
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

File: frontend/src/components/Footer/Footer.jsx [hunk 5]
```diff
@@ -1,98 +1,71 @@
...
+          <Link
+            href={item.link}
+            target="_blank"
+            rel="noopener noreferrer"
```
Comment (desoindx) [inline frontend/src/components/Footer/Footer.jsx:54]: not for all :/

File: frontend/src/components/Header/Header.jsx [hunk 6]
```diff
@@ -1,90 +1,91 @@
...
+} from '@dataesr/react-dsfr';
 import { useStore } from 'stores/';
 
-import Menu from './Menu';
-import TopMenu from './TopMenu';
+const defaultItems = [
+  { title: 'Comment ça se passe ?', link: '/' },
+  { title: 'Trouver un psychologue', link: '/trouver-un-psychologue' },
+  { title: 'Foire aux questions', link: '/faq' },
+  // { title: 'Nous contacter', link: '/mentions-legales' },
```
Comment (desoindx) [inline frontend/src/components/Header/Header.jsx:22]: better to remove it totally

File: frontend/src/components/Header/Header.jsx [hunk 7]
```diff
@@ -1,90 +1,91 @@
...
+  { title: 'Remboursement de mes séances', link: '/psychologue/mes-remboursements' },
+  { title: 'Mes informations', link: '/psychologue/mon-profil' },
+];
 
 const Header = () => {
-  const [open, setOpen] = useState(false);
   const location = useLocation();
-  const { userStore: { user } } = useStore();
+  const { userStore: { user, deleteToken } } = useStore();
+  const [path, setPath] = useState(() => location.pathname || '');
```
Comment (desoindx) [inline frontend/src/components/Header/Header.jsx:35]: why not always use location.pathname ?

File: frontend/src/components/Footer/Footer.jsx (hunk 4 above)
Comment (desoindx) [reply frontend/src/components/Footer/Footer.jsx:50]: and if you do, you can use 
.map((item, index) => {})

File: frontend/src/components/Header/Skiplinks.jsx [hunk 8]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Skiplinks as DSSkiplinks, SkiplinkItem } from '@dataesr/react-dsfr';
+
+const Skiplinks = () => (
+  <DSSkiplinks>
+    <SkiplinkItem href="#contenu">Contenu</SkiplinkItem>
+    <SkiplinkItem href="#header-navigation">Menu</SkiplinkItem>
```
Comment (desoindx) [inline frontend/src/components/Header/Skiplinks.jsx:7]: I don't think we have a header-navigation ?

Comment (sandrica89) [reply frontend/src/components/Header/Skiplinks.jsx:7]: Yes, it's done by react-dsfr

**Review by desoindx — APPROVED** [review]

**Pull request merged**